app/views/lectures/show.html.erb

```diff
@@ -21,14 +21,16 @@
 <div class="tab-content" id="lectureTabsContent">
   <div class="tab-pane fade show active mx-auto" id="dashboard" role="tabpanel" aria-labelledby="dashboard-tab" style="max-width: 2000px;">
     <div class="row">
-      <div id="updates" class="col-12 col-sm-6 col-xl-4">
-          <% if @current_user.is_student %>
-            <h2 class="h5">Updates</h2>
-          <% else %>
-            <h2 class="h5">Questions</h2>
-          <% end %>
-          <%= react_component("UpdatesApp", {user_id: @current_user.id, is_student: @current_user.is_student, lecture_id: @lecture.id, course_id: @lecture.course.id, interactions_enabled: @lecture.allow_interactions?, questions_list: Question.questions_for_lecture(@lecture, current_user)}) %>
-      </div>
+      <% if @current_user.is_student %>
+        <div id="updates" class="col-12 col-sm-6 col-xl-4">
+            <% if @current_user.is_student || @lecture.questions_enabled %>
+              <h2 class="h5">Updates</h2>
+            <% elsif @lecture.questions_enabled %>
+              <h2 class="h5">Questions</h2>
```

Comment:
This is never reached due to if condition...

Reply:
Question will be shown if the user is not a student and questions are enabled. I don't see why this shouldn't work? Could you explain it a bit more detailed please?

Reply:
Yes, is now fixed with the newest change.

Reply:
It was because of the wrapping is_student condition right?

Reply:
Exactly. :)